test/integration/reason/local.bs.js

```diff
@@ -0,0 +1,11 @@
+// Generated by BUCKLESCRIPT VERSION 1.9.2, PLEASE EDIT WITH CARE
```

Comment:
doesn't seem like this should be committed...

Reply:
I hard coded the result of compiling the Reason file with BuckleScript so that we don’t have to run it when running the tests. BuckleScript is a binary that compiles itself during installation which could take a while. By hard coding the result we can test just the loading logic, because the logic to run BuckleScript is contained in another package, bsb-js.